cypress/support/commandsCreateSignal.js

```diff
@@ -24,6 +25,28 @@ export const checkDescriptionPage = () => {
   cy.contains('Geef het tijdstip aan').should('be.visible');
   cy.contains("Foto's toevoegen").should('be.visible');
   cy.contains('Voeg een foto toe om de situatie te verduidelijken').should('be.visible');
+  cy.get('body').then($body => {
+    if ($body.find(`${CREATE_SIGNAL.disclaimer}`).length > 0) {
+      checkHeaderFooter();
+    }
+  });
+};
+
+export const checkFlashingOrange = () => {
+  cy.get('.highlight--active')
```

Comment:
There's a pending change that will no longer have this class name available on the element. The element, however, will be targetable by the attribute `data-testid="highlight"` and, when active, with class name `animate`.